rpc/src/client/transport/websocket.rs

```diff
@@ -1034,8 +1034,17 @@ mod test {
         event_tx: ChannelTx<Event>,
     }
 
+    // A setting telling which of the CometBFT server versions to emulate
+    // with the test server.
+    #[derive(Copy, Clone)]
+    enum TestRpcVersion {
+        V0_34,
+        V0_37,
+        V0_38,
+    }
```

Comment:
It's almost like `CompatMode` could be given the new `V0_38` variant as well, but it's avoidable on the client side and I think it would result in lots of unnecessary code duplication.